src/components/modern/Button.js

```diff
@@ -71,10 +134,9 @@ const TouchableContainer = styled(TouchableOpacity)`
 
 const Title = styled(Text)`
   ${fontStyles.medium};
-  ${({ theme, $variant }) => ($variant === 'primary' && `color: ${theme.colors.buttonPrimaryTitle};`)};
-  ${({ theme, $variant }) => ($variant === 'primary-destructive' && `color: ${theme.colors.buttonPrimaryTitle};`)};
-  ${({ theme, $variant }) => ($variant === 'secondary' && `color: ${theme.colors.buttonSecondaryTitle};`)};
-  ${({ theme, $variant }) => ($variant === 'text' && `color: ${theme.colors.buttonTextTitle};`)};
-  ${({ theme, $variant }) => ($variant === 'text-destructive' && `color: ${theme.colors.negative};`)};
   ${({ $size }) => $size === 'large' && `font-family: ${appFont.medium}`};
 `;
+
+const LeftIcon = styled(Icon)`
+  margin-top: 1px;
```

Comment:
Just thinking, it gets cut or what? 🤔

Reply:
Attempt at pixel-perfect layout. Reviewed that again on iOS & Android and I decided to remove it as not really needed any more.